src/rust/engine/src/context.rs

```diff
@@ -273,9 +273,13 @@ impl Core {
     local_cache_read: bool,
     local_cache_write: bool,
   ) -> Result<Arc<dyn CommandRunner>, String> {
-    // TODO: Until we can deprecate letting the flag default, we implicitly disable
-    // eager_fetch when remote execution is in use. See the TODO in `global_options.py`.
-    let eager_fetch = remoting_opts.cache_eager_fetch && !remoting_opts.execution_enable;
+    // TODO: Until we can deprecate letting the flag default, we implicitly default
+    // cache_content_behavior when remote execution is in use. See the TODO in `global_options.py`.
```

Comment:
Should this be documented in help text for the option?

Reply:
The deprecation is already in place.